packages/selected-network-controller/src/SelectedNetworkController.ts

```diff
@@ -4,9 +4,11 @@ import type {
   BlockTrackerProxy,
   NetworkClientId,
   NetworkControllerGetNetworkClientByIdAction,
+  NetworkControllerGetStateAction,
   NetworkControllerStateChangeEvent,
   ProviderProxy,
 } from '@metamask/network-controller';
+import type { HasPermissions } from '@metamask/permission-controller';
```

Comment:
nit. not important, just feel that it reads more obviously when it's used later in this file

```suggestion
import type { HasPermissions as PermissionControllerHasPermissions } from '@metamask/permission-controller';
```

Reply:
removed here: https://github.com/MetaMask/core/commit/ecb5e72a77608459747c279a2e14aad4c94b63a1